README.md

```diff
@@ -196,8 +196,17 @@ for documentation.
 
 To start processing jobs, you need to start a separate process
 (in additional to your Rails app) with `bin/aws_active_job_sqs`
-(an executable script provided with this gem).  You need to specify the queue to
-process jobs from:
+(an executable script provided with this gem). You can poll for one or multiple
+queues using the `--queue` argument(s). You can  specify multiple queues 
```

Comment:
nit: double spacing between "can" and "specify"

Reply:
Also could start a new line on multiple queues for ease of readability (personal preference, however)